back-end-side/Models/Record.cs

```diff
@@ -17,5 +17,16 @@ public class Record
 
         [Ignore]
         public string? Category { get; set; } = "other";
+
+        public int CompareTo(object? obj)
+        {
+            if (obj == null) return 1;
+
+            Record nextRecord = obj as Record;
+            if (nextRecord != null)
+                return this.Date.CompareTo(nextRecord.Date);
+            else
+                throw new ArgumentException("exeption");
```

Comment:
exception should be clearer, not just "exception"

Reply:
K.